content/react/concepts/hooks/hooks.md

```diff
@@ -28,48 +28,3 @@ Hooks are imported at the top of a file from the `react` library:
 ```jsx
 import React, { useState, useEffect } from 'react';
```

Comment:
```suggestion
import React, { useHook } from 'react';
```

Reply:
Hmm, do you think this may mislead readers into thinking there's an actual hook called `useHook`?

Reply:
@krowter I see your point, but I also think its important to make sure this is in line with what a \```pseudo code block should look/be like in the "Syntax" section. How about we attach a **Note** that explains this point? Maybe something like the following?

> \> \*\*Note:\*\* The code snippet above is purely pseudocode and `useHook` is not an actual React hook.